### ISSUE TITLE
System User fails to import on initial startup

### ISSUE THREAD

**Issue by wezell** [dotCMS/core#6884]
On starting a fresh dotCMS (master-3.1) it seems that the ESAPI validator code is preventing the system user and the anonymous users from getting inserted.  This is running against a new mysql db.

```
SecurityConfiguration for Logger.LogApplicationName not found in ESAPI.properties. Using default: true
SecurityConfiguration for Logger.LogServerIP not found in ESAPI.properties. Using default: true
2015-01-02 11:36:13,611 WARN  IntrusionDetector - [SECURITY FAILURE Anonymous:null@unknown -> /dotcms/IntrusionDetector] Invalid input: context=firstName, type(UserName)=^[a-zA-Z0-9_.\-]*$, input=system user
com.dotcms.repackage.org.owasp.esapi.errors.ValidationException: firstName: Invalid input. Please conform to regex ^[a-zA-Z0-9_.\-]*$ with a maximum length of 50
	at com.dotcms.repackage.org.owasp.esapi.reference.validation.StringValidationRule.checkWhitelist(StringValidationRule.java:144)
	at com.dotcms.repackage.org.owasp.esapi.reference.validation.StringValidationRule.checkWhitelist(StringValidationRule.java:160)
	at com.dotcms.repackage.org.owasp.esapi.reference.validation.StringValidationRule.getValid(StringValidationRule.java:284)
	at com.dotcms.repackage.org.owasp.esapi.reference.DefaultValidator.getValidInput(DefaultValidator.java:214)
	at com.dotcms.repackage.org.owasp.esapi.reference.DefaultValidator.isValidInput(DefaultValidator.java:152)
	at com.dotcms.repackage.org.owasp.esapi.reference.DefaultValidator.isValidInput(DefaultValidator.java:143)
	at com.liferay.portal.ejb.UserLocalManagerImpl.validate(UserLocalManagerImpl.java:617)
	at com.liferay.portal.ejb.UserLocalManagerImpl.updateUser(UserLocalManagerImpl.java:415)
	at com.liferay.portal.ejb.UserLocalManagerUtil.updateUser(UserLocalManagerUtil.java:203)
	at com.dotmarketing.business.UserFactoryLiferayImpl.saveUser(UserFactoryLiferayImpl.java:249)
	at com.dotmarketing.business.UserAPIImpl.getSystemUser(UserAPIImpl.java:135)
	at com.dotmarketing.util.ImportExportUtil.doXMLFileImport(ImportExportUtil.java:1359)
	at com.dotmarketing.util.ImportExportUtil.doXMLFileImport(ImportExportUtil.java:1075)
	at com.dotmarketing.util.ImportExportUtil.doImport(ImportExportUtil.java:409)
	at com.dotmarketing.db.DotCMSInitDb.loadStarterSite(DotCMSInitDb.java:99)
	at com.dotmarketing.db.DotCMSInitDb.InitializeDb(DotCMSInitDb.java:40)
	at com.dotmarketing.startup.runalways.Task00004LoadStarter.executeUpgrade(Task00004LoadStarter.java:15)
	at com.dotmarketing.startup.StartupTasksExecutor.executeUpgrades(StartupTasksExecutor.java:214)
```

**Comment by jtesser** [comment]
strange i don't think I had that issue locally.  We'll take a look.  I don't think it would be DB related

**Comment by jcastro-dotcms** [comment]
Pull request: https://github.com/dotCMS/core/pull/6885

**Comment by bryanboza** [comment]
Fixed, tested on master-3.1 // H2 // FF.... New case opened here #6886
